packages/components/psammead-brand/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 5.1.0-alpha.8 | [PR#2833](https://github.com/bbc/psammead/pull/2833) Wrap `scriptLink` in a div to fix Read&Write issue in chrome |
```

Comment:
We have two entries for in changelog for `5.1.0-alpha.8`, I think we should bump this to `5.1.0-alpha.9` to rectify this

Reply:
Done.